tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -48,6 +57,17 @@ const userSharesFileOrFolderWithUser = function (file, sharee, role) {
 const userSharesFileOrFolderWithGroup = function (file, sharee, role) {
   return userSharesFileOrFolderWithUserOrGroup(file, sharee, true, role)
 }
+
+Given('user {string} from remote server has shared {string} with user {string} from local server', function (sharer, file, receiver) {
+  receiver = `${receiver}@${httpHelper.getBackendUrl('LOCAL')}/`
+  return httpHelper.runOnRemoteServer(shareFileFolder, [file, sharer, receiver, SHARE_TYPES.federated_cloud_share])
```

Comment:
I don't know how i feel about sending an array of args. `.call()` does the same, but would love to hear other options.